content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1114,15 +1114,15 @@ used to determine if the user has applied settings different from the kubeadm de
 
 Used on: Node
 
-Label that kubeadm applies on the control plane nodes that it manages.
+Label that kubeadm applies on the control plane nodes that it manages. If this label set, [EndpointSlice controller](/docs/concepts/services-networking/topology-aware-routing/#implementation-control-plane) ignores that node.
 
 ### node-role.kubernetes.io/control-plane {#node-role-kubernetes-io-control-plane-taint}
 
 Used on: Node
 
 Example: `node-role.kubernetes.io/control-plane:NoSchedule`
 
-Taint that kubeadm applies on control plane nodes to allow only critical workloads to schedule on them.
+Taint that kubeadm applies on control plane nodes to allow only critical workloads to schedule on them. If this taint set, EndpointSlice controller ignores that node.
```

Comment:
Which does the EndpointSlice controller check for? The taint, the label, or neither?

Reply:
Hi @sftim,
With the same confusion as you, i just added it on both label and taint. and waiting for someone to comment on it to know the correct behaviour.